webpack.config.js

```diff
@@ -1,33 +1,10 @@
 const path = require("path");
 const webpack = require("webpack");
 const CopyPlugin = require("copy-webpack-plugin");
-const TerserPlugin = require("terser-webpack-plugin");
 const CompressionPlugin = require("compression-webpack-plugin");
-const HttpProxyMiddlware = require("http-proxy-middleware");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
-const dfxJson = require("./dfx.json");
 require("dotenv").config();
 
-/** Read the replica host from dfx.json */
-function readReplicaHost() {
```

Comment:
Not necessary anymore with these changes

Reply:
I _guess_ we could still use it in the [/api proxy](https://github.com/dfinity/internet-identity/pull/877/files#diff-1fb26bc12ac780c7ad7325730ed09fc4c2c3d757c276c3dacc44bfe20faf166fL116), but it wasn't used there until now, so I'm happy removing it